pkg/manager/member/tidb_member_manager_test.go

```diff
@@ -224,6 +224,41 @@ func TestTiDBMemberManagerSyncUpdate(t *testing.T) {
 				g.Expect(set.Spec.Template.Spec.Containers).To(HaveLen(2))
 			},
 		},
+		{
+			name: "scale TiDB to 0 did not clear failureMembers",
```

Comment:
It is better to use a behavior description as the case name, e.g. TiDB should clear failureMembers when scale to 0